src/app/core/pipes/format-datetime.pipe.spec.ts

```diff
@@ -1,100 +1,48 @@
-import { createPipeFactory, mockProvider, SpectatorPipe } from '@ngneat/spectator/jest';
-import { Store } from '@ngrx/store';
-import { of } from 'rxjs';
+import { createPipeFactory, SpectatorPipe } from '@ngneat/spectator/jest';
+import { utcToZonedTime } from 'date-fns-tz';
 import { FormatDateTimePipe } from 'app/core/pipes/format-datetime.pipe';
 
 describe('FormatDateTimePipe', () => {
   let spectator: SpectatorPipe<FormatDateTimePipe>;
-  const inputValue = 1632831778445;
+  const inputValue = utcToZonedTime(1709604204732 + (11 * 1 * 60 * 60 * 1000), 'Europe/London').getTime();
```

Comment:
I don't quite understand the conversion here and why it is better than a simple timestamp.